packages/flutter_tools/lib/src/localizations/gen_l10n_types.dart

```diff
@@ -422,13 +416,13 @@ class AppResourceBundle {
     String localeString = resources['@@locale'] as String;
 
     // Look for the first instance of an ISO 639-1 language code, matching exactly.
-    final String fileName = globals.fs.path.basenameWithoutExtension(file.path);
+    final String fileName = file.fileSystem.path.basenameWithoutExtension(file.path);
```

Comment:
Was able to get rid of `globals` by pulling the filesystem off the passed in `file`.